sklearn/metrics/_plot/confusion_matrix.py

```diff
@@ -70,6 +71,9 @@ def plot(self, *, include_values=True, cmap='viridis',
             Axes object to plot on. If `None`, a new figure and axes is
             created.
 
+        colorbar : matplotlib colorbar, default=True
+            If `False` colorbar to the plot is not added.
```

Comment:
Maybe say "Whether to add a colorbar to the plot"?

Reply:
Thanks @amueller for reviewing. I will work on those suggestions.